diffRules/rules.json

```diff
@@ -0,0 +1,33 @@
+[
```

Comment:
What is this file? is it a default of some sort? or example? and should we name it appropriately to convey that?

Reply:
This file has the rules defined that runs with update APIs.

Reply:
I think default might be a better name for it then.  Or mercury.  Just trying to distinguish it more than calling it "rules"

Reply:
I will rename to **defaultRules.json**